content/en/boilerplates/gateway-api-version.md

```diff
@@ -0,0 +1,5 @@
+---
+---
+{{< text syntax=plain snip_id=gateway_api_version >}}
```

Comment:
nit: the generated env var name is pretty ugly `bpsnip_gateway_api_version_gateway_api_version`

How about: `snip_id=value` instead?

Reply:
Done.